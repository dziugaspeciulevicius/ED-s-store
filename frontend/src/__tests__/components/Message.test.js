import React from "react";
import { shallow } from "enzyme";
import "../../setupTests";
import Message from "../../components/Message.component";
jest.mock("../../components/Message.component");

describe("Message tests", () => {
    it("Renders <Message /> component correctly", () => {
      // shallow creates instance of a component
      expect(shallow(<Message />)).toMatchSnapshot();
    });
});
