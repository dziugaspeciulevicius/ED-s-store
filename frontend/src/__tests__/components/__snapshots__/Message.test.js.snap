// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Message tests Renders <Message /> component correctly 1`] = `ShallowWrapper {}`;
